### PR TITLE
ascheck 0.1.0 (new formula)

### PR DESCRIPTION
**Pull request by harryzcy**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/ascheck.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  url "https://github.com/harryzcy/ascheck/archive/refs/tags/v0.1.0.tar.gz"
+  sha256 "5a90fa2b521ca68fced0866a8055c36fb7367a37bd8d04ac5e3ad391b6706fa4"
+  license "MIT"
+  head "https://github.com/harryzcy/ascheck.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-o", "ascheck"
+    bin.install "ascheck"
```
Comment (carlocab) [inline Formula/ascheck.rb:13]:
```suggestion
    system "go", "build", *std_go_args
```

File: Formula/ascheck.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  end
+
+  test do
+    system "#{bin}/ascheck", "--help"
```
Comment (carlocab) [inline Formula/ascheck.rb:17]: Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

**Review by carlocab — COMMENTED** [review]
Thanks for your PR, @harryzcy. This looks like a nice piece of software, but it currently does not meet the requirements for inclusion into homebrew/core. The CI log reads
```
ascheck:
  * GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)
Error: 1 problem in 1 formula detected
```
You are welcome to submit this again when it meets this requirement. Please address my comments below before re-submitting.

If you wish, you can host this formula in your own tap. Here are some docs to help you get started:

https://docs.brew.sh/Taps
https://docs.brew.sh/Interesting-Taps-and-Forks
https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap
https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/

File: Formula/ascheck.rb (hunk 2 above)
Comment (harryzcy) [reply Formula/ascheck.rb:17]: Sure, thank you.

**Pull request closed without merging**